### ISSUE TITLE
transfer nasbnation static components in middleman

### ISSUE THREAD

**Issue by joshRpowell** [milner-group/nasbnation#1]
**Background**
The zipfile below contains the views and assets for www.nasbnation.com . I did my best to remove all unnecessary files, however you may find some unnecessary stylesheets and javascripts related to other parts of the site.  The stylesheets and javascripts can be found under site.scss and app-site.

Zipfile: [nasbnation.zip](https://github.com/milner-group/nasbnation/files/706572/nasbnation.zip)

**Goal**
1. Set-up the static site components in a Middleman app.  
2. Re-organize files, optimize images, etc
3. Determine best implementation for forms
4. Set-up analytics 
5. Set-up log-in

In summary, we want to port the existing site to Middleman as is, with little or no changes. 
Once we have isolated the site in Middleman and tested in a staging environment we will need
to modify the log-in process.  Any additional changes / modifications will be made once the site is relaunched. 

All of the styling is in bootstrap.

**Comment by joshRpowell** [comment]
I forgot that public assets were in a different folder here: 
[public_assets.zip](https://github.com/milner-group/nasbnation/files/708456/public_assets.zip)
